tests/test_apel_plugin.py

```diff
@@ -1,5 +1,9 @@
 import pytest
 from apel_plugin import functions
```

Comment:
> functions isn't a great name for a sub module.

I agree, any suggestions?

> It is better to import the needed functions individually because it makes it much clearer what is imported from where.

All of my functions are in `functions`, so it would look like
```
from apel_plugin import functions.regex_dict_lookup
from apel_plugin import functions.get_begin_previous_month
...
```
and so on. 
Is this what you have in mind?

Reply:
Done with commit [918029b](https://github.com/ALU-Schumacher/AUDITOR-APEL-plugin/pull/22/commits/918029bee6819577b72d7fc94d57534c70619149)